test/autotest.py

```diff
@@ -525,9 +527,11 @@ def testRestart():
       if i.endswith(".dmtcp"):
         cmd+= " "+ckptDir+"/"+i
     #run restart and test if it worked
-    procs.append(launch(cmd))
+    procs.append(runCmd(cmd))
     WAITFOR(lambda: doesStatusSatisfy(getStatus(), status),
             wfMsg("restart error"))
+    #wait and see if process will die shortly after restart
+    sleep(S*SLOW)
```

Comment:
We need to insert `CHECK(doesStatusSatisfy(getStatus(), status))` after this sleep to make it effective.